HackYourFuture-CPH/Guide-IT
# added the files for burger menu

# Description

If the resolution goes below 1024 we can make the menu into a burger menu.

Fixes # (issue)
#117 
# How to test?

open the home page in the browser by giving the npm run dev command in the editor then goto developer tools under the 1024px resolution then you can able to see the burger menu.
# Checklist
